components/ui/unstyled-button.tsx

```diff
@@ -0,0 +1,18 @@
+import type { ComponentProps } from 'react';
+
+export function UnstyledButton(props: ComponentProps<'button'>) {
+  return (
+    <button
+      style={{
+        background: 'none',
+        border: 'none',
+        color: 'inherit',
+        font: 'inherit',
+        margin: 0,
+        padding: 0,
+      }}
+      type={props.type ?? 'button'}
```

Comment:
not sure how to get pass this linter warning...

Reply:
how about giving button as default only without props.type? Since if we pass type props next line will override.
```suggestion
      type='button'
```